gamehub-frontend/src/types.ts

```diff
@@ -19,6 +19,57 @@ export type Game = {
     coverImage: string
 }
 
+export type GameAPIResponse = {
+    count: number
+    next: string,
+    pervious: string,
+    games: GameAPI[],
+}
+
+export type GameAPI = {
+    id: string,
+    title: string,
+    genre: string[],
+    releaseDate: string,
+    platforms: string[],
+    coverImage: string
+}
+
+export type AddDeleteLibrary = {
```

Comment:
The name of type is a little bit confusing. It is for adding library? or deleting library?
Isn't it better to make it more clear, for example AddDeletedLibrary or..? (for your future colleagues :) )

Reply:
Thats the fun part. its for both :D

Reply:
okay, just asked ChatGPT. It suggested like
- UserGameLibraryAction
- LibraryModification

but it's totally up to you :)